lisa/tools/fdisk.py

```diff
@@ -83,6 +83,18 @@ def delete_partitions(self, disk_name: str) -> None:
                 sudo=True,
             )
 
+    def change_partition_type(self, disk_name: str, type_code: str) -> None:
+        # t -> change a partition type
+        self.node.execute(
+            f"(echo n; echo p; echo 1; echo ; echo ; echo t;"
+            " echo {type_code}; echo p; echo w) | "
+            f"{self.command} -w always -W always {disk_name}",
```

Comment:
-w -W not supported in older fdisk version

Reply:
should we just remove them? in our test case it's new disk and new partition seems wipe signature is not 'necessary'?